dipy/workflows/base.py

```diff
@@ -101,19 +101,33 @@ def add_workflow(self, workflow):
             ref_text = [text if text else "\n" for text in npds['References']]
             ref_idx = self.epilog.find('References: \n') + len('References: \n')
             self.epilog = "{0}{1}\n{2}".format(self.epilog[:ref_idx],
-                                               ''.join([text for text in ref_text]),
+                                        ''.join([text for text in ref_text]),
                                                self.epilog[ref_idx:])
 
         self.outputs = [param for param in npds['Parameters'] if
                         'out_' in param[0]]
 
         args, defaults = get_args_default(workflow.run)
 
+        output_args = \
+            self.add_argument_group('output arguments(optional)')
```

Comment:
This fits on one line (col 76)

Reply:
Hey,
Thanks for the update. I updated the code to adjust in a single line.